application/features/Term.py

```
@@ -118,12 +131,15 @@
 if not app.debug or os.environ.get("WERKZEUG_RUN_MAIN") == "true":
     TERMINAL_PORT = find_free_port()
     print("TERMINAL_PORT =", TERMINAL_PORT)
+    logger.debug("Term: Terminal port {}".format(TERMINAL_PORT))
```

Ruff (0.8.2)

Use f-string instead of `format` call

Convert to f-string

(UP032)